package calculatorTest;

import static org.junit.Assert.*;

import org.junit.Rule;
import org.junit.Test;
import org.junit.rules.ExpectedException;

import Exception.IllegalCharacterException;
import Exception.OutstepException;
import panel.App;
/*
public class jumpSubjectTest {
	App app = new App();
	@Rule
	public ExpectedException exception = ExpectedException.none();
	@Test//没启动
	public void test1() throws IllegalCharacterException, OutstepException {
		app.jumpSubject("q");
	}
	@Test//非法字符
	public void test2() throws IllegalCharacterException, OutstepException {
		app.Generate("4","4");
		exception.expect(IllegalCharacterException.class);
		app.jumpSubject("q");
	}
	@Test//越界
	public void test3() throws IllegalCharacterException, OutstepException {
		app.Generate("4","4");
		exception.expect(OutstepException.class);
		app.jumpSubject("5");
	}
	@Test//正常情况
	public void test4() throws IllegalCharacterException, OutstepException {
		app.Generate("4","4");
		app.jumpSubject("4");
		assertEquals(4,app.mark);
	}
	}
*/

